addons/mail/static/src/core/common/composer.xml

```diff
@@ -89,6 +89,7 @@
                     attachments="props.composer.attachments"
                     unlinkAttachment="attachmentUploader.unlink"
                     imagesHeight="50"/>
+                <EmojiPicker t-if="ui.isSmall and state.keyboard === KEYBOARD.EMOJI" className="'o-mail-Composer-bottomPicker w-100'" onSelect="(str) => this.addEmoji(str)"/>
```

Comment:
I would say, don't make a mobile view here so that the code can be less redundant with https://github.com/odoo/odoo/pull/123671 and seems this is not the scope of gif picker. ( And can be painful to rebase 😢 )

Reply:
But ok, if this is already been done, then keep it so the PR can be quickly merged. And if this function is not fully developed then we don't have to put top much time into implementing this function.